src/notifications/notifications.service.ts

```diff
@@ -0,0 +1,40 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Comment } from 'src/entity/comment.entity';
+import { Image } from 'src/entity/image.entity';
+import { Notification } from 'src/entity/notification.entity';
+import { User } from 'src/entity/user.entity';
+import { sendImageEmailNotification } from 'src/helpers/email.helper';
+import { Repository } from 'typeorm';
+
+@Injectable()
+export class NotificationsService {
+  constructor(
+    @InjectRepository(User)
+    private readonly usersRepository: Repository<User>,
+    @InjectRepository(Notification)
+    private readonly notificationRepository: Repository<Notification>,
+    @InjectRepository(Comment)
+    private readonly commentRepository: Repository<Comment>,
+  ) {}
+
+  async imageUploaded(image: Image) {
+    try {
+      await this.notificationRepository.save({
+        type: 'image',
```

Comment:
We should use typescript type to limit options for "type" of notifications. Now, it can be added anything, lets say type: 'something' and compiler will not complain about that.